### PR TITLE
Closes #429: Persistent object ids - change from int to long

### LINKED ISSUE #429: Persistent object ids - change from int to long
See #380

### PR DESCRIPTION
**Pull request by lukdumi**

### CODE REVIEW

File: saos-common/src/test/java/pl/edu/icm/saos/common/testcommon/IntToLongMatcher.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        return new Long(actualIntValue).equals(expectedLongValue);  
+    }  
+  
+    @Override  
+    public void describeTo(Description description) {  
+        description.appendValue(actualIntValue)  
+                .appendText(" has been found instead of ")  
+                .appendValue(expectedLongValue);  
+    }  
+}
```
Comment (pawelczak) [inline saos-common/src/test/java/pl/edu/icm/saos/common/testcommon/IntToLongMatcher.java]: Missing newline.

**Comment by pawelczak** [comment]
You forgot about class JudgmentCriteria.

**Comment by pawelczak** [comment]
There are two classes, that need int -> long changes: CcListController, ScListController.

**Comment by lukdumi** [comment]
:+1:

**Pull request closed without merging**